src/features/calendar/components/CalendarWeekView/index.tsx

```diff
@@ -111,7 +111,11 @@ const CalendarWeekView = ({ focusDate, onClickDay }: CalendarWeekViewProps) => {
         {/* Hours column */}
         <Box>
           {range(24).map((hour: number) => {
-            const time = dayjs().set('hour', hour).set('minute', 0);
+            const time = dayjs().set('hour', hour).set('minute', 0).toString();
+            const is12HourClock = Intl.DateTimeFormat(navigator.language, {
```

Comment:
...and that this logic can be removed :)

Reply:
I was trying to change width to other boxes except for the very parent box! Very easy and nice solution! Thank you! ❤️